examples/whisper/requirements.txt

```diff
@@ -1,6 +1,5 @@
 neural-compressor
 onnx==1.14.0
-onnxruntime>=1.15.0
```

Comment:
the onnxruntime version requirements are still needed so that users can run the examples independently.

Reply:
the ORT is installed in pipeline stage. If we add it, the ort night installation will be overwritten.